src/sidebar/store/modules/annotations.js

```diff
@@ -255,7 +272,9 @@ const actions = util.actionTypes(reducers);
 function addAnnotations(annotations) {
   return function (dispatch, getState) {
     const added = annotations.filter(annot => {
-      return !findByID(getState().annotations.annotations, annot.id);
+      return (
+        !annot.id || !findByID(getState().annotations.annotations, annot.id)
+      );
```

Comment:
Is this a "just because you were here" change or is it more meaningful?

Reply:
This code was previously incorrect, as it did not account for `addAnnotations` being passed an unsaved annotation. As far as I can tell the effect of this mistake would be to not apply the anchoring timeout when a new unsaved annotation was added to the store, but only if there were already other unsaved annotations. This is an edge case that would be very hard to test manually as it would require that an annotation the user has just created to fail to anchor in the same document.